pkgs/top-level/aliases.nix

```
@@ -1422,6 +1422,7 @@ mapAliases {
   ### Y ###
 
   yacc = throw "'yacc' has been renamed to/replaced by 'bison'"; # Converted to throw 2024-10-17
+  yesplaymusic = throw "YesPlayMusic has been removed as it was broken, unmaintained, and used deprecated Node and Electron versions"; # Added 2024-12-13
```

```suggestion
  yesplaymusic = throw "YesPlayMusic has been removed as it was broken, unmaintained in Nixpkgs, and used deprecated Node and Electron versions"; # Added 2024-12-13
```
We should specify "in Nixpkgs" here as this can be read as upstream no longer being maintained -- which they [seem to be ](https://github.com/qier222/YesPlayMusic/commit/022009b140df0897386be7c63b3c8014433cf954), but correct me if I'm wrong

---

Well, it actually doesn't work anymore (there's nearly 600 open issues without any response), and I think the version of the CloudMusic API it's using stopped working entirely some time ago, since the *web* version is down as well...

---

Ah this is fine then